### PR TITLE
Update DG Documentation 

### PR DESCRIPTION
**Pull request by kumsssss**
1) Updated Appendix: Instructions for Manual Testing section.

### CODE REVIEW

**Review by chewwh09 — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -990,118 +990,130 @@ testers are expected to do more *exploratory* testing.
...
+     Expected: All items that have already expired for at least 3 weeks are shown.
 
 ### Sorting items
 
 1. Sorting items in the inventory in terms of quantity.
 
-    1. Test case: `sort quantity asc`<br>
-       Expected: All the items are sorted in terms of quantity from least to most.
+    * Test case: `sort quantity asc`<br>
+       Expected: All the items are sorted in ascending quantity.
```
Comment (Md-Fazil) [inline docs/DeveloperGuide.md]: Should this be in order of ascending quantity?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -990,118 +990,130 @@ testers are expected to do more *exploratory* testing.
...
 
-    1. Test case: `sort quantity desc`<br>
-       Expected: All the items are sorted in terms of quantity from most to least.
+    * Test case: `sort quantity desc`<br>
+       Expected: All the items are sorted in descending quantity.
```
Comment (Md-Fazil) [inline docs/DeveloperGuide.md]: Should this be in order of descending quantity?

**Review by Md-Fazil — APPROVED** [review]
Some suggestions to consider. Otherwise, LGTM!!

**Review by github-amanda — APPROVED** [review]
LGTM!

**Pull request merged**